lib/psci/psci_common.c

```diff
@@ -332,7 +332,7 @@ void psci_get_parent_pwr_domain_nodes(unsigned int cpu_idx,
 				      unsigned int node_index[])
 {
 	unsigned int parent_node = psci_cpu_pd_nodes[cpu_idx].parent_node;
-	int i;
+	uint32_t i;
```

Comment:
minor: Prefer not to use a mix of `uint32_t` and `unsigned int` for `parent_node` and `i`. The TF coding guidelines suggest using the latter, although I know that MISRA prefers the former.